pkg/test/echo/common/model.go

```diff
@@ -18,6 +18,7 @@ import "istio.io/istio/pkg/config/protocol"
 
 // TLSSettings defines TLS configuration for Echo server
 type TLSSettings struct {
+	//RootCert supplies the root certificate to use inside the echo server, will use the default system cert pool if this is empty.
```

Comment:
Missed whitespace after `//`, and also see the other comment to see if this should be updated.

Reply:
I edited the code following the 2 option. Thanks a lot!

Reply:
this looks better but still not what you're doing in the code. In your code you call NewCertPool if RootCert is empty, this means you're not always appending to the system cert pool.

If you want to make this always append, then just always call SystemCertPool() in the other place, there is no need to call NewCertPool() any more, and also change the comment to  `// If not empty, RootCert supplies the extra root cert that will be appended to the system cert pool.`